### PR TITLE
GitHub support resources file

### PR DESCRIPTION
**Pull request by SeanFeldman**
Adding [GH support file](https://github.com/blog/2400-support-file-support).
Might need to go through some changes.
Please review.

### CODE REVIEW

File: .github/SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Support resources to check prior to raising issues
+
+1. Azure Service Bus [samples](https://github.com/Azure/azure-service-bus/tree/master/samples)
+1. [StackOverflow](https://stackoverflow.com/questions/tagged/azureservicebus)
```
Comment (vinaysurya) [inline .github/SUPPORT.md]: >1. [](start = 0, length = 2)

nit: numbering?

Comment (SeanFeldman) [inline .github/SUPPORT.md]: What about it? 🙂

File: .github/SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+1. Already [resolved issues](https://github.com/Azure/azure-service-bus-dotnet/issues?q=is%3Aissue+is%3Aclosed)
```
Comment (vinaysurya) [inline .github/SUPPORT.md]: >. Already [resolved issues](https://github.com/Azure/azure-service-bus-dotnet/issues?q=is%3Aissue+is%3Aclosed) [](start = 1, length = 110)

nit: move this up before stackoverflow perhaps?

**Comment by vinaysurya** [comment]
@SeanFeldman, Thanks for adding this! I was thinking may be we could add a section in the README.md to say "How to get support" and that would be described as "See our Support Guidelines" and carry a link to this md file?

**Comment by SeanFeldman** [comment]
@vinaysurya not a bad idea. I'll add it.

**Comment by SeanFeldman** [comment]
@vinaysurya updated and rebased to resolve branch differences.

**Comment by SeanFeldman** [comment]
@vinaysurya you forgot something 😛 
image
Clicked it for you 😃

**Pull request merged**